Geometry/ForwardCommonData/data/hfnose/v0/hfnose.xml

```diff
@@ -122,9 +122,13 @@
     <Numeric name="Sectors"          value="36"/>
     <Vector name="SlopeBottom" type="numeric" nEntries="2">
       [HFNoseSlope1], [HFNoseSlope1]</Vector>
+    <Vector name="ZFrontBottom" type="numeric" nEntries="2">
```

Comment:
@bsunanda - CMS policy is to version the xml files. This helps to keep track of the scenarios which will need to be modified: 2023D31 in this case. Please, create a v2 of hfnose.xml and remove the v0 which is not used and the v1 if it will not be used. Thanks.

Reply:
@ianna the changes are due to a change in the DDHGCalEEAlgo which was needed for getting 2023D32 working. At that time it slipped out of my mind to update the HFNose geometry xml files which also need the same code. As a result D31 was not operational at all. I can make v32 - but then has to delete v31 as well and modify Configuration/Geometry and Geometry/CMSCommonData. This is not a new version - simple fix due to change in the algorithm code

Reply:
ok, but v0 is not used in any of the scenarios. Do you still need it?

Reply:
It is used in a standalone test - not for general scenario - but useful without the rest of CMS